.github/auto_assign.yml

```diff
@@ -7,14 +7,9 @@ addAssignees: false
 # A list of reviewers to be added to pull requests (GitHub user name)
 reviewers:
   - akkomar
```

Comment:
It might be a good idea to add some new DE here.

Reply:
Split to https://github.com/mozilla-services/mozilla-pipeline-schemas/issues/714